### PR TITLE
Combined users challenge finished

### PR DESCRIPTION
**Pull request by uetrozi**

### CODE REVIEW

File: css-basics/css-basics_combined-users/css/styles.css [hunk 1]
```diff
@@ -46,23 +46,47 @@ li {
...
+  padding: 12px;
+  border-radius: 4px;
+}
 .name {
   font-size: 18px;
   text-transform: uppercase;
   margin: 0;
 }
 
+aside .name {
```
Comment (CheckersRT) [inline css-basics/css-basics_combined-users/css/styles.css:68]: maybe this needs to be `aside .user .name` to specify the .name's in the .user's?

File: css-basics/css-basics_combined-users/css/styles.css [hunk 2]
```diff
@@ -46,23 +46,47 @@ li {
...
+  font-size: 16px;
+}
 .bio {
   font-size: 14px;
   color: darkslategray;
 }
 
+aside .bio {
```
Comment (CheckersRT) [inline css-basics/css-basics_combined-users/css/styles.css:76]: `aside .user .bio`? Maybe it doesn't make any difference here.

File: css-basics/css-basics_combined-users/css/styles.css [hunk 3]
```diff
@@ -46,23 +46,47 @@ li {
...
+  margin: 6px 0;
+}
+
 .location {
   font-size: 12px;
   font-weight: 600;
   text-align: end;
   margin: 0;
 }
+aside .location {
```
Comment (CheckersRT) [inline css-basics/css-basics_combined-users/css/styles.css:87]: `aside .user .location`

**Review by CheckersRT — COMMENTED** [review]
🚀

File: css-basics/css-basics_combined-users/css/styles.css (hunk 1 above)
Comment (uetrozi) [reply css-basics/css-basics_combined-users/css/styles.css:68]: I know what you mean. But both ways have the same end result so i'll just leave it like this coz i'm lazy.

File: css-basics/css-basics_combined-users/css/styles.css (hunk 2 above)
Comment (uetrozi) [reply css-basics/css-basics_combined-users/css/styles.css:76]: 🤷‍♀️ same

File: css-basics/css-basics_combined-users/css/styles.css (hunk 3 above)
Comment (uetrozi) [reply css-basics/css-basics_combined-users/css/styles.css:87]: same

**Pull request merged**